guacamole/src/main/frontend/src/app/client/controllers/clientController.js

```diff
@@ -485,17 +498,23 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
         $scope.menu.connectionParameters = newFocusedClient ?
             ManagedClient.getArgumentModel(newFocusedClient) : {};
 
+        // Re-broadcast the updated client
+        $scope.$broadcast('guacClientChanged'. newFocusedClient);
```

Comment:
Looks like that `.` was meant as a comma here.

Reply:
Oops, yeah. I had fixed this but forgot to push the commit...